INSTALL.md

```diff
@@ -1,19 +1,20 @@
 # Table of Contents
 
+- [Table of Contents](#table-of-contents)
 - [Installing and Deploying](#installing-and-deploying)
   - [Recommended Approach](#recommended-approach)
-  - [Local Setup on Windows](#local-setup-on-windows)
-  - [Local Setup using Docker (Recommended)](#local-setup-using-docker-recommended)
+  - [Local setup on Windows](#local-setup-on-windows)
+  - [Local setup using Docker (Recommended)](#local-setup-using-docker-recommended)
     - [Build your own docker image](#build-your-own-docker-image)
-  - [Local Setup (Legacy)](#local-setup-legacy)
+  - [Local Setup with VSCode Remote Containers](#local-setup-with-vscode-remote-containers)
```

Comment:
Please add again `Local Setup (Legacy, no longer supported)` in the toc

Reply:
Again a good catch. Also the text "with VSCode Remote Containers" was wrong.
Now updated automatically using the  devcontainer markdown extension

Reply:
I use GitHub copilot, so it does this part for me.